howtos/release-procedure.md

```diff
@@ -21,20 +21,51 @@ The build process for [publish-prod](https://github.com/camunda/camunda-platform
 
 ## Perform a minor release
 
-Minor releases to Camunda Platform 8 happen twice a year in April and October.
+Minor releases to Camunda Platform 8 happen twice a year in April and October, and the documentation is versioned on the same cadence.
 
-To prepare for a minor release, you'll need to create a new verison.
+To prepare for a minor release, you'll need to create a new version.
 
 ### Create new version
 
-Technically, the current contents are frozen in `docs` and copied to `versioned_docs` with the corresponding version. The process can be triggered by this Docusaurus command:
+The versioning process copies a snapshot of the current documentation from the un-versioned source to a new versioned source location.
```

Comment:
I like this and I think it makes the most sense. Should we provide an example at this point that shows in the folder structure what is changing? Or would that be too noisy and unnecessary?

Reply:
what do you think about linking to https://github.com/camunda/camunda-platform-docs/blob/5a2cbf3b0628d1cbb89071f15c40e5652be84f29/howtos/versioning.md#structure? I feel like that table captures things properly, and it seems complicated enough to have one source of truth instead of duplicating.

Reply:
Oooooh yes!

Reply:
check out https://github.com/camunda/camunda-platform-docs/pull/1334/commits/10b59ea09d82310feb9e0b297dbec112cb3054ec! It's not in the exact spot of this comment but I think it works.